packages/toolkit/.changes/next-release/Bug Fix-e5a76bd7-06a7-4a59-aead-8ea3e3924ee9.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "region in debug config should override default toolkit region"
```

always helpful to customers to mention exact field names. also, markdown is supported.

also, https://github.com/aws/aws-toolkit-vscode/pull/6011 is about `aws.credentials` , not regions? If there are multiple issues being fixed, it's definitely worth creating multiple changelog items.

```suggestion
	"description": "SAM local debugging: `aws.region` specified in launch config should override default Toolkit region"
```

---

Sure, updating